document/service.py

```diff
@@ -53,7 +83,6 @@ def get_document_by_id(self, doc_id: str) -> DocumentResponse | None:
 
     def create_document(self, request: DocumentCreate):
         request.validate()
-        self.logger.debug(f"model dump: {request.model_dump()}" )
```

Comment:
any particular reason for erasing this line? I think it's helpful for debugging

Reply:
Ah yes, i accidentally remove it